src/Main.java

```diff
@@ -1,5 +1,317 @@
+        //Ukoliko upisete y, ponovno unosite brojeve koji cete dodavati na niz
+        // kada ponovno unesete -10 ispisu vam se svi brojevi koje ste unjeli.
+
+        System.out.println("\n");
+        System.out.println("Task 2");
+        List<Integer> listOfNumbers = new ArrayList<Integer>();
+        int numberOfArray = 0;
+        while (numberOfArray != -10) {
+            System.out.println("Enter the number of array: ");
+            numberOfArray = reader.nextInt();
+            if (numberOfArray != -10)
+                listOfNumbers.add(numberOfArray);
+        }
+
+        System.out.println("Numbers of array are: ");
+        System.out.println(listOfNumbers);
+
+        System.out.println("Would you like to add more numbers into array ? Y / N");
+        String addMoreItems = reader.next();
+        if (addMoreItems.equals("Y")) {
+            numberOfArray = 0;
+            while (numberOfArray != -10) {
+                System.out.println("Add new number to existing array: ");
+                numberOfArray = reader.nextInt();
+                if (numberOfArray != -10)
+                    listOfNumbers.add(numberOfArray);
+            }
+
+            System.out.println("Numbers of array are: ");
+            System.out.println(listOfNumbers);
+        }
+        //3. Napravide funkciju koja provjereava da li je nesto palindrom,
+        //Napravite Funkciju koja iz niza izbacuje brojeve koji mi specifikujemo,
+        //Napravite funkciju koja provjerava da li je broj djeljiv sa brojem x.
+        //Kada se pokrene aplikacija Pojavi vam se meni
+        //
+        //-Is string a palindrom
+        //-Filter an Array
+        //-Number is divisible by X.
+        //Ukoliko izaberete  broj 2
+        //Morate unjeti prvo brojeve za niz i zatim broj koji zelite izbaciti iz niza…
+        System.out.println("\n");
+        System.out.println("Task 3");
+        System.out.println("Chose an option - fill in the optipon No. 1, 2 or 3? ");
+        System.out.println("\n");
+        System.out.println(" - Is string a palindrom - opcija 1");
+        System.out.println(" - Number is divisible by X - opcija 2 ");
+        System.out.println(" - Filter an Array - opcija 3 ");
+        int option = reader.nextInt();
+        Task3 task3 = new Task3();
+
+        switch (option) {
+            case 1:
+                //Is string a palindrom
+                System.out.println("Fill in the word you want to check if it is a palindrom ? ");
+                String word = reader.next();
+                boolean isPalindrom = task3.IsPalindrom(word);
+                if (isPalindrom == true) {
+                    System.out.println("The word is palindrome!");
+                } else
+                    System.out.println("The word is not a palindrome!");
+
+                break;
+            case 2:
+                //-Number is divisible by X.
+                System.out.println("Enter wanted number ?");
```

Comment:
again switch-case
and you have parts of logic in case instead of function to contain the whole logic....